examples/jiralert.yml

```diff
@@ -4,6 +4,10 @@ defaults:
   api_url: https://jiralert.atlassian.net
   user: jiralert
   password: 'JIRAlert'
+  # optional client cert
+  # cert_file: /path/to/cert.crt
+  # key_file: /path/to/cert.key
+  # insecure_skip_verify: false
```

Comment:
This should be default, so let's remove it

Reply:
Do we want to indicate that insecure_skip_verify is an option in the config, e.g.,

    # insecure_skip_verify: true

?